tfhe/src/shortint/list_compression/compressed_server_keys.rs

```diff
@@ -132,3 +134,47 @@ impl ClientKey {
         (glwe_compression_key, glwe_decompression_key)
     }
 }
+
+impl ParameterSetConformant for CompressedCompressionKey {
+    type ParameterSet = CompressionParameters2;
+
+    fn is_conformant(&self, parameter_set: &Self::ParameterSet) -> bool {
+        let Self {
+            packing_key_switching_key,
+            lwe_per_glwe,
+            storage_log_modulus,
+        } = self;
+
+        let params = PackingKeyswitchConformanceParams {
+            decomp_base_log: parameter_set.packing_ks_base_log,
+            decomp_level_count: parameter_set.packing_ks_level,
+            input_lwe_dimension: parameter_set
+                .standard_glwe_dimension
```

Comment:
standard ?

Reply:
In the classical AP, before and after decompression
Do you have a suggestion for a better name?

Reply:
ah, you mean the dimension we compress from the AP ?

i.e.

AP(n) -> compress(some other n) ?

Reply:
No, it's `AP(standard_params) -> Compressed(packing_ks_params) -> AP(standard_params)`

Reply:
got it, could be "uncompressed_lwe_dimension" and "compressed_whatever" this way we know what we refer to

Reply:
Renamed `uncompressed_`